requirements-dev.txt

```diff
@@ -0,0 +1,20 @@
+# Install with `pip install -r requirements-dev.txt`
+
+# Developer tooling
+autoflake
+black
+isort
+pytest
+pytest-cov
+pytest-xdist
+pylint
+pre-commit
+mypy
+tox
+
+# Documentation tooling
+sphinx
+m2r
+
```

Comment:
Everything above already came in, with #108 .
The remaining two lines appear to be new, however.